Sources/Common/CustomerIOInstance.swift

```diff
@@ -135,7 +71,7 @@ public class CustomerIO: CustomerIOInstance {
         // arising from lifecycle differences between wrapper SDKs and native SDK.
         let globalDataStore = diGraph.globalDataStore
         if let token = globalDataStore.pushDeviceToken {
-            registerDeviceToken(token)
+            // registerDeviceToken(token)
```

Comment:
`postInitialize` is already being removed in `MBL-123` so this shouldn't be a problem